src/components/Quiz/Quiz.stories.tsx

```
@@ -0,0 +1,355 @@
+              areaKnowledgeId: 'matematica',
+              subjectId: 'geometria',
+              topicId: 'angulos',
+              subtopicId: 'internos',
+              contentId: 'geometria'
+            }
+          ],
+          options: [
+            { id: 'opt1', option: '90°' },
+            { id: 'opt2', option: '180°' },
+            { id: 'opt3', option: '180°' },
+            { id: 'opt4', option: '360°' }
```

ESLint

[error] Insert `,`

(prettier/prettier)